exporter/honeycombmarkerexporter/config.go

```diff
@@ -24,33 +28,37 @@ type Config struct {
 
 	// Markers is the list of markers to create
 	Markers []Marker `mapstructure:"markers"`
+
+	confighttp.HTTPClientSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	exporterhelper.QueueSettings  `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings  `mapstructure:"retry_on_failure"`
 }
 
 type Marker struct {
-	// Type defines the type of Marker.  Markers with the same type appear in Honeycomb with the same color
+	// Type defines the type of Marker.
 	Type string `mapstructure:"type"`
 
-	// Color is the color of the Marker. Will only be used if the Type does not already exist.
-	Color string `mapstructure:"color"`
-
-	// MessageField is the attribute that will be used as the message.
+	// MessageKey is the attribute that will be used as the message.
 	// If necessary the value will be converted to a string.
-	MessageField string `mapstructure:"message_field"`
+	MessageKey string `mapstructure:"message_key"`
 
-	// URLField is the attribute that will be used as the url.
+	// URLKey is the attribute that will be used as the url.
 	// If necessary the value will be converted to a string.
-	URLField string `mapstructure:"url_field"`
+	URLKey string `mapstructure:"url_key"`
 
 	// Rules are the OTTL rules that determine when a piece of telemetry should be turned into a Marker
 	Rules Rules `mapstructure:"rules"`
+
+	// DatasetSlug is the endpoint that specifies the Honeycomb environment
+	DatasetSlug string `mapstructure:"datasetslug"`
 }
 
 type Rules struct {
-	// ResourceConditions is the list of ottlresource conditions that determine a match
-	ResourceConditions []string `mapstructure:"resource_conditions"`
-
 	// LogConditions is the list of ottllog conditions that determine a match
 	LogConditions []string `mapstructure:"log_conditions"`
+
+	//
+	logBoolExpr expr.BoolExpr[ottllog.TransformContext]
 }
 
 var defaultCfg = createDefaultConfig().(*Config)
```

Comment:
I missed this in the first PR, but we do not need to set the config in `Validate`.  If the cfg is nil thats an err